### PR TITLE
documenting new Mappings API endpoints

### PR DESCRIPTION
**Pull request by austenschunk-okta**
## Description:
- Adding documentation for the Public Mapping API endpoints: list Mappings, get Mapping, partial update Mapping. 
- Targeted for monolith release: 2019.08.0

### Resolves:

* [OKTA-236299](https://oktainc.atlassian.net/browse/OKTA-236299)

### Reviewers:
@colinwang-okta @donwoods-okta @jakubvul-okta @okta/ud-core

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/mappings/index.md [hunk 1]
```diff
@@ -0,0 +1,536 @@
+---
+title: Mappings
+category: management
+---
+
+# Mappings API
+
+The Okta Mappings API provides operations to manage the mapping of properties between an Okta User's and an App User's
+[Profile Properties](/docs/reference/api/users/#profile-object) using [Expression Lanugage](/docs/reference/okta-expression-language).
+More information on *Okta User* and *App User* Profiles can be found in
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/docs/reference/api/mappings/index.md]: Please remove all italics.

File: packages/@okta/vuepress-site/docs/reference/api/mappings/index.md [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+document with the following properties :
+
+| Property    | Description                                                                              | DataType                                                                     | Nullable | Unique | Readonly | Validation  |
+|:------------|:-----------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------|:---------|:-------|:---------|:------------|
+| expression  | combination or single source properties that will be mapped to the target property       | [Expression Language Object](/docs/reference/okta-expression-language/)      | FALSE    | FALSE  | FALSE    |             |
+| pushStatus  | whether to update target properties on user create & update or just on create            | `DONT_PUSH` or `PUSH`                                                        | FALSE    | FALSE  | FALSE    |             |
+
+> Having a pushStatus of `PUSH` will cause properties to be in the target to be updated on create and update.
```
Comment (colinwang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/mappings/index.md]: this sentences seems to be off to be

Comment (jakubvul) [reply packages/@okta/vuepress-site/docs/reference/api/mappings/index.md]: > Having a `pushStatus` value of `PUSH` will cause properties in the target to be updated on User create and update. 

?

Comment (austenschunk-okta) [reply packages/@okta/vuepress-site/docs/reference/api/mappings/index.md]: Yeah, I'll change it to that.

**Pull request merged**